### ISSUE TITLE
I created a quest, but I'm not doing any damage whatsover

### ISSUE THREAD

**Issue by Xanduba** [HabitRPG/habitica#7078]
General Info
- UUID:  caca5af7-f980-4f0f-8ff4-b3df7f667f15
- Browser: Firefox and the game app on mobile
- OS: Windows and Android
### Description

[//](#): #  I am not doing any damage to the monster from the quest base-list that I created (its more than 40 hours since it began). I did my dailies (some hard, some avarage, some easy), habits and to-dos. I don't know whats the problem or how to describe it more than that!

**Comment by ghost** [comment]
Did you use the Inn last night? When you are resting in the inn, you do not do damage; it is saved until the next time your day rolls over after leaving the inn. 

Also, have you refreshed the party page? (Sometimes it just doesn't show up.) 

If neither of those are the problem, please report back.

**Comment by Alys** [comment]
@Xanduba From what I can see in the database, you are doing damage to the quest boss, so it looks like one of Cantras's suggestions would have been correct. 

I'm closing this now but if you still have problems or have any questions, comment again here! Give us answers for Cantras's questions if you're able to. We'll still see your reply even though this is closed.

**Comment by Xanduba** [comment]
Hello, thank you for your prompt support. The story is a little too long, but I'll try to make it short:
- I didnt use the Inn; It wasnt a refresh-and-done thing.
  the story:
- 3 days ago I quit my old party (with an open quest-invitation there) and joined this new party, where I hosted a new quest. When I did so, it popped up an error message telling me that I should reload the website. When I did, it came back with the old quest invitation (another monster, the same from the previous party), and I denied it. It popped-up the same error, and as I reloaded the website, it was alright with the quest ON.
  Then it happened: 35 hours were past and I didnt do any damage. when I looked the quest participants, it named the old party members.
  So I guessed it somehow bugged with those 2 quests together, and maybe my damage was going to the old party's quest.

so I canceled the quest and started anew yesterday. Now its everything right and smooth.
